generators/app/index.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable consistent-return */
+const chalk = require('chalk');
+const AppGenerator = require('generator-jhipster/generators/app');
+// const jhipsterPackagejs = require('generator-jhipster/package.json');
+
+module.exports = class extends AppGenerator {
+    // /**
+    //  * Override yeoman standard storage function for yo-rc.json
+    //  *  in order to save variables in generator-jhipster key.
+    //  * @return {String} The name of the root generator
+    //  */
+    // rootGeneratorName() {
+    //     return jhipsterPackagejs.name;
```

Comment:
This is good to have to re-use certain generator properties from the JHipster generator. For now, this should be okay.